telemetry/definitions/commonDefinitions.json

```diff
@@ -7138,6 +7138,26 @@
                     "type": "result"
                 }
             ]
+        },
+        {
+            "name": "webview_error",
```

Comment:
Why is this needed? Every event can store `result`, `reason`, `reasonDesc`. Errors should not be modeled as separate events, they should be modeled as existing events with `result=Failed` + `reason`.

E.g. in this case webview_load could be such an event. If failure can occur after load, and we want a generic "webview interaction" event, then let's consider that instead. E.g. `webview_click`, `webview_submit`, etc.

Reply:
webview_error in this case is an generic error that can get thrown inside of mynah ui. Not sure if you can tie it directly to a chatMessage or anything since it can appear whenever: https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/amazonq/webview/ui/main.ts#L26

Reply:
see https://github.com/aws/aws-toolkit-common/pull/839#issuecomment-2332486730